LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 HUMAN Security
```

Comment:
```suggestion
Copyright (c) 2024 HUMAN Security.
```

Reply:
ugh sorry, fixed 🙏 
i'm closing this one, we can open another as needed